syntax = "proto3";

package goya.rpc.echo;

option cc_generic_services = true;

message EchoRequest {
  string message = 1;
}

message EchoResponse {
  string message = 1;
}

service EchoServer {
  rpc Echo(EchoRequest) returns(EchoResponse);
}

